tests/manage/cluster/cluster_expansion/test_node_expansion.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from ocs_ci.utility.utils import TimeoutSampler
+from tests import helpers
+from ocs_ci.framework.testlib import tier1, ManageTest
+from ocs_ci.ocs import machine as machine_utils
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.node import wait_for_nodes_status
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+class TestAddNode(ManageTest):
+    def test_add_node(self):
+        before_replica_counts = dict()
+        count = 2
+        machines = machine_utils.get_machinesets()
+        for machine in machines:
+            before_replica_counts.update({machine: machine_utils.get_replica_count(machine)})
+        worker_nodes_before = helpers.get_worker_nodes()
+        logger.info(f'before count of worker nodes {worker_nodes_before}')
```

Comment:
suggesting to change to:

f'The worker nodes number before adding a new node is {worker_nodes_before}'